network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -54,17 +54,22 @@ <h1 class="h1-heading">{{ page.title }}</h1>
     {% endif %}
   {% endfor %}
 </div>
-<div class="pt-1 pt-md-2 pb-4 pb-md-5 col-lg-8 m-auto bottom-share-button-container">
-  <div class="share-button-group-wrapper mt-5 d-print-none"
-    data-share-text="{% blocktrans with title=page.title %}{{ title }} by @mozilla{% endblocktrans %}"
-    data-link="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}"></div>
 
-  {% if show_comments %}
-    <h2 class="h4-heading">{% trans "Comments" %}</h2>
-    <div id="commento"></div>
-  {% endif %}
+<div class="container">
```

Comment:
Since Modular page was wrapping a container and row div around blog page/oppurtunity page/campaign page templates, their files have been updated to accomodate for the removal of those original wrapping divs